columnflow/tasks/framework/base.py

```diff
@@ -1266,15 +1359,14 @@ def single_repr(values: Sequence[str]) -> str:
         return _repr + f"_{law.util.create_hash(all_values)}"
 
     @classmethod
-    def broadcast_to_configs(cls, params: dict[str, Any], param: str, n_config_insts: int) -> tuple[Any]:
-        value = params.get(param, law.no_value)
-        if not isinstance(value, tuple):
+    def broadcast_to_configs(cls, value: Any, name: str, n_config_insts: int) -> tuple[Any]:
+        if value == () or not isinstance(value, tuple):
```

Comment:
Minor optional change for readability.

```suggestion
        if not isinstance(value, tuple) or not value:
```